src/v/config/configuration.cc

```diff
@@ -1254,6 +1254,13 @@ configuration::configuration()
        .visibility = visibility::tunable},
       2,
       {.min = 1})
+  , debug_load_slice_warning_depth(
+      *this,
+      "debug_load_slice_warning_depth",
+      "The recursion depth after which debug logging will be enabled "
```

Comment:
The recursion depth after which debug logging is enabled automatically for the log reader.

Reply:
^^ @Deflaimun Looks like this was merged before we could review. Please update in the docs/add to our PR to update the source for the node properties.

Reply:
We're probably going to remove this. It isn't meant to be changed by users.